package.json

```diff
@@ -76,6 +77,16 @@
       "presets": [
         "microsoft/react-native"
       ],
+      "lint": {
+        "lockfile": {
+          "noDuplicates": {
+            "packages": [
+              "#react-native",
```

Comment:
for knowledge what's "#" mean?

Reply:
I think it's custom syntax, and similar to an align-deps profile. It was introduced here: https://github.com/microsoft/rnx-kit/pull/3652/files